Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,59 @@
-import React from 'react';
+import React, { Component } from 'react';
 import './App.css';
+import { CalculatorDisplay } from './components/DisplayComponents/CalculatorDisplay';
 
-const App = () => {
-  return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
-  );
-};
-
-export default App;
+export class App extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			total: 0,
+			value: ''
+		};
+	}
+
+	clearValue() {
+		this.setState(prevState => ({
+			value: prevState.total
+		}));
+	}
+
+	displayValue(inputValue) {
+		this.setState(prevState => ({
+			value: `${(prevState.value += inputValue)}`
+		}));
+	}
+
+	handleClick = evt => {
+		const value = evt.target.dataset.value;
+
+		switch (value) {
```

Comment:
Seems a bit overkill because you have only 1 conditional but good to see you are trying new thing!